src/qt/walletframe.cpp

```diff
@@ -215,6 +215,14 @@ void WalletFrame::gotoLoadPSBT(bool from_clipboard)
         }
         std::ifstream in{filename.toLocal8Bit().data(), std::ios::binary};
         data.assign(std::istream_iterator<unsigned char>{in}, {});
+
+        // Some psbt files may be base64 strings in the file rather than binary data
```

Comment:
I dislike this kind of ambiguity in file formats, it's a thing that can bite you in unexpected ways at some point, but I understand the reason. A valid binary PSBT can never be valid b64, I hope?

Reply:
> A valid binary PSBT can never be valid b64, I hope?

Yes. The magic in binary are the characters `psbt`, but are `cHn...` in base64. Interpreting the binary as base64 or vice versa would result in the wrong magic.